addons/block_code/ui/blocks/utilities/drag_drop_area/drag_drop_area.gd

```diff
@@ -1,16 +1,60 @@
 @tool
-extends MarginContainer
+## Drag drop area.
+##
+## A Control which watches for click and drag gestures beginning from itself.
+## It propagates events up to its parent, so it is possible to place this
+## control inside a control which processes input events such as [LineEdit].
+## If a drag occurs, it emits [signal drag_started].
+extends Control
```

Comment:
Any reason to change this from a `MarginContainer`? Maybe the margin was pointless in this case since we just want the rectangle for collecting input events.

Reply:
Yeah, it just didn't quite vibe with me. I think in an earlier version of this we were actually putting things inside of a DragDropArea? At this point we aren't.